app/Models/Movie.php

```diff
@@ -32,4 +32,118 @@ public function celebs() {
+            'https://www.youtube.com/watch?v=5JsZ2bAl3Jk&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=35',
+            'https://www.youtube.com/watch?v=fgqEyC19538&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=41',
+            'https://www.youtube.com/watch?v=mYmQkxiqn20&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=45',
+            'https://www.youtube.com/watch?v=CHT8era7GOk&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=54',
+            'https://www.youtube.com/watch?v=dvaycRwzsxA&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=64',
+            'https://www.youtube.com/watch?v=WhLTyBUpUmg&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=61',
+            'https://www.youtube.com/watch?v=_dikcPOJBqk&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=67',
+            'https://www.youtube.com/watch?v=gjs6lKhedwg&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=72',
+            'https://www.youtube.com/watch?v=06g6kHWNQoE&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=75',
+            'https://www.youtube.com/watch?v=8ibK_JRC7Xw&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=88',
+            'https://www.youtube.com/watch?v=RSXc5z9F7Do&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=96',
+            'https://www.youtube.com/watch?v=5upIB9kxdvM&list=PLuAiHxLeTqiTeCoAiB39PUYALbxKprq6e&index=106',
+        ];
+
+        return $trailers[array_rand($trailers)];
+    }
+
+    public static function randomBanner()
+    {
+        $banners = [
+            'https://wallpapercave.com/wp/wp5223134.jpg',
+            'https://wallpapercave.com/wp/wp7999745.jpg',
+            'https://wallpaperaccess.com/full/889517.jpg',
+            'https://wallpapercave.com/wp/wp3666962.jpg',
+            'https://wallpapercave.com/wp/wp5539841.jpg',
+            'https://images.wallpapersden.com/image/download/joaquin-phoenix-as-joker_a2lma2uUmZqaraWkpJRnamtlrWZpaWU.jpg',
+            'https://images.hdqwalls.com/wallpapers/a-quiet-place-part-2-movie-n3.jpg',
+            'https://wallpapercave.com/wp/wp4817824.jpg',
+            'https://wallpaperaccess.com/full/1264684.jpg',
+            'https://images.hdqwalls.com/wallpapers/up-movie-4k-fr.jpg',
+            'https://trumpwallpapers.com/wp-content/uploads/WALL-E-Wallpaper-01-1920x1080-1.jpg',
+            'https://i.pinimg.com/originals/f0/3a/2b/f03a2bcaf5c64e81aa6c494ffe98be6e.jpg',
+            'https://cdn.hipwallpaper.com/i/8/44/hSHi6N.jpg',
+            'https://wallpaperaccess.com/full/1707195.jpg',
+            'https://studioghiblimovies.com/wp-content/uploads/2019/02/d78b1219f9991c69a7619d362548cb79f4d29e02_hq2.jpg',
+            'https://wallpaperaccess.com/full/4190641.jpg',
+            'https://wallpapercave.com/wp/wp3083904.jpg',
+            'https://images2.alphacoders.com/742/thumb-1920-742320.png',
+            'https://free4kwallpapers.com/uploads/originals/2019/11/20/bladerunner--movie-scene-wallpaper.jpg'
+        ];
```

Comment:
Jayd, maybe these functions would make more sense if it was on the seeding? This is basically the only time we will use it right?

Reply:
Like what you have done with genre? What do you think?

Reply:
Hi @vrodriguero! Okay I'll try that then!

Reply:
Hi @vrodriguero, I moved it to the factory instead since I found it simpler than having to create a seeder. What do you think? Is it best practice to do it in the seeder? I just can't figure out how to use the seeder for it but let me know if its preferred to use a seeder and I'll take some time to look online if that's alright?